content/guides/servicediscovery.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Guide to Service Discovery with Docker
+kind: guide
+listorder: 10
+---
+Docker is being adopted rapidly, and for good reason: it simplifies many aspects of running a service in production. But Docker-powered services typically run many more containers than traditional services run hosts, so monitoring is much more complex. Now with platforms like Kubernetes and ECS orchestrating your containers, you may not even know on what host your containers are running—which makes monitoring containers and the images they run even more complex.
+
+How can you monitor a service which is always hopping from one host to another?
+
+Datadog automatically keeps track of what is running where, thanks to its Service Discovery feature.
+
+It allows you to define configuration templates for specific images in a distributed configuration store on top of the Datadog Agent which will use them to dynamically reconfigure its checks when your containers ecosystem changes. Whether you use Kubernetes, Amazon ECS, or Docker Swarm to manage your Docker containers, you can now monitor images, such as NGINX or Redis, even if containers running them stopped or new ones started, without interruption or having to restart the Agent.
+
+## How does it work?
+
+The Service Discovery feature is continuously listening to Docker events and spots those triggered when a container is created, destroyed, started or stopped. When one of these happens, the Agent identifies which service is impacted, loads the configuration template you will have defined for this image, and automatically set up its checks according to the new disposition.
+
+Configuration templates are defined in a single key-value store per cluster. We are currently supporting etcd and Consul. Zookeeper will be added very soon.
```

Comment:
"Configuration templates are defined in a single key-value store per cluster. We currently support etcd and Consul."

Minor tweaks above. Lets not mention ZK since it's not yet supported.